lib/src/main/java/com/thenewboston/data/dto/bankapi/bankdto/response/Banks.kt

```diff
@@ -27,7 +27,7 @@ data class Bank(
     @SerialName("node_identifier")
     val nodeIdentifier: String,
     @SerialName("port")
-    val port: String? = null,
+    val port: Int? = null,
     @SerialName("protocol")
     val protocol: String,
     @SerialName("version")
```

Comment:
Can we add the missing `version` property from the response as well please.

http://143.110.137.54/banks to see the full response.

Reply:
![image](https://user-images.githubusercontent.com/50385769/102555577-0ef25780-4095-11eb-9eff-327639dc9c2f.png)
Version isn't missing